src/index.ts

```diff
@@ -27,15 +27,23 @@ export default class BitField {
      * @default 0.
      */
     private readonly grow: number;
+
     /** The internal storage of the bitfield. */
     public buffer: Uint8Array;
+
     /** The number of bits in the bitfield. */
     get length(): number {
-        return this.buffer.length >> 3;
+        return this.buffer.length << 3;
     }
 
+    /** Variable to store the count of bits set to 1 */
+    private setBitCount: number = 0;
+
+    /** Set to keep track of indices of set bits */
+    private readonly setBits: Set<number> = new Set();
+
```

Comment:
I would like to keep the memory usage of bitfield the same as it is, and especially the set will require quite a bit more memory. They are also not needed for `isEmpty`. Let's remove them?